todo_app/data/mongodb_items.py

```diff
@@ -0,0 +1,55 @@
+from pymongo import MongoClient
+import os
+
+def connect_to_mongo():
```

Comment:
This is a good solution for this exercise, but worth noting that this creates a fresh connection every time which is why you may notice your app is reasonably slow. A simple adjustment to speed this up might be a basic singleton pattern:
```suggestion
client = None
def connect_to_mongo():
    global client
    if client is not None:
        return client[os.getenv('MONGODB_DATABASE_NAME')]
```